base/reshapedarray.jl

```diff
@@ -223,21 +225,17 @@ end
     val
 end
 
-@inline function _unsafe_setindex!(A::ReshapedArray, val, indexes::Int...)
+@inline function _unsafe_setindex!(A::ReshapedArray{T,N}, val, indexes::Vararg{Int,N}) where {T,N}
     @inbounds parent(A)[ind2sub_rs(A.mi, sub2ind(size(A), indexes...))...] = val
     val
 end
-@inline function _unsafe_setindex!(A::ReshapedArrayLF, val, indexes::Int...)
```

Comment:
Generic array handling infrastructure will call `sub2ind` for us. (This was just a "stale" method that I happened to notice while working on this.)